src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="./index.html">
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo image"
+            class="header__logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="./index.html"
+                class="is-active nav__link"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="./index.html"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav__item">
```